### DISCUSSION TITLE
PowerShell Download URL

### DISCUSSION THREAD

**Discussion by mengelsen** [PowerShell/PowerShell#17778] [Q&A]
I am trying to automate software deployment using our MDM solution. Does this project have a download URL that would allow me to consistently pull the latest (stable) version of the PowerShell (.pkg or .exe) installer?

I am looking for a dynamic URL that looks something like this:

Examples:
VS Code (Download latest version): [https://code.visualstudio.com/sha/downlo...=win32-x64](https://code.visualstudio.com/sha/download?build=stable&os=win32-x64)
Asana (Download latest version): [https://desktop-downloads.asana.com/win3...aSetup.exe](https://desktop-downloads.asana.com/win32_x64/prod/latest/AsanaSetup.exe)
BOX Tools (Download latest version): [https://e3.boxcdn.net/box-installers/box...nstall.msi](https://e3.boxcdn.net/box-installers/boxedit/win/currentrelease/BoxToolsInstaller-AdminInstall.msi)

Thanks!

**Comment by mnaray** [comment]
I found some PowerShell modules/packages on here, I think one or multiple of these might be what you're looking for.

https://www.powershellgallery.com/packages

Let me know if you need help with anything else.

**Comment by kasini3000** [comment]
i was write a powershell script to install newest win pwsh from github, take a look and copy code you needed,i wish it can help you.
https://gitee.com/chuanjiao10/kasini3000/blob/master/zkj_install_k%E8%AE%B0%E9%A5%AD%E5%BA%97_win%E4%B8%BB%E6%8E%A7%E6%9C%BA.ps1

**Comment by kilasuit** [comment]
Currently no, however it is something that could be implemented in the not too distant future and is perhaps something for @SteveL-MSFT & team to consider

**Comment by doctordns** [comment]
Can't you use the install-powershell.ps1 script???
